src/analyzer.ts

```diff
@@ -502,20 +503,18 @@ export class StreamLoader implements UrlLoader {
   }
 
   // We can't load external dependencies.
-  canLoad(url: string): boolean {
-    return this._buildAnalyzer.analyzer.canResolveUrl(url);
+  canLoad(url: ResolvedUrl): boolean {
+    return url.startsWith('file://');
```

Comment:
Nit: recommend one more slash like `file:///` to specifically cover hostless urls

Reply:
Updated.